PraceDomowe/PracaDomowa5/PD5_Oskar_Werner/PD5_Oskar_Werner.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
<p>Ciekawe rozwiązanie, ale z drobnym błędem: liczba pytań w zbiorze to nie są tylko te Posts dla których istnieje IsAccepted ma wartość True, lecz te, których PostTypeId == 1 (chociaż to trzeba by było odczytać z opisu danych).</p><p> </p><p><code>2 punkty</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>